core/playwright.config.ts

```diff
@@ -17,6 +17,8 @@ export default defineConfig({
 	expect: {
 		timeout: process.env.CI ? 5_000 : 60_000,
 	},
+	// don't continue going after 3 tests have failed, that's too many
+	maxFailures: process.env.CI ? 3 : undefined,
```

Comment:
maybe controversial. since every single test was failing and retrying 3 times, i'd rather know after 3 tests fail that i'm cooked already

Reply:
Good change!